2nd-gen/time-server/functions/index.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * TODO: license
+ */
+'use strict';
+
+// [START functionsimport]
```

Comment:
I *think* we decided to prepend all the v2 tag names with "v2," is that correct Jeff?

So this one would be [START v2functionsimport]

It definitely will be helpful if the snippet CLI passes that naming into the final snippet name, which I think it will, and will test when the first v2 sample gets merged.

Reply:
yes, let's prepend with v2